release/rancher/rancher.go

```diff
@@ -671,34 +671,50 @@ func GenerateAnnounceReleaseMessage(ctx context.Context, ghClient *github.Client
 		return "", errors.New("release commit sha is nil")
 	}
 
-	isPreRelease := strings.ContainsRune(tag, '-')
-	// if preRelease
-	// get the rancher-components.txt artifact and extract images with RC and components with RC
-	//  else
-	// go to rancher/rancher/package/Dockerfile and get the UI and CLI versions
-
 	r := releaseAnnnouncement{
 		Tag:              tag,
 		PreviousTag:      previousTag,
 		RancherRepoOwner: rancherRepoOwner,
 		CommitSHA:        *release.TargetCommitish,
 		ActionRunID:      actionRunID,
-		ImagesWithRC: []string{
-			"rancher/aks-operator v1.9.2-rc.1",
-			"rancher/cis-operator v1.0.15-rc.2",
-		},
-		ComponentsWithRC: []string{
-			"SYSTEM_AGENT_VERSION v0.3.9-rc.4",
-			"WINS_AGENT_VERSION v0.4.18-rc2",
-		},
-		UIVersion:  "v2.9.2-ui",
-		CLIVersion: "v2.9.2-cli",
 	}
 
 	announceTemplate := announceReleaseGATemplate
+
+	// only prerelease images like alpha, rc or hotfix contain - (e.g: v2.9.2-alpha1)
+	isPreRelease := strings.ContainsRune(tag, '-')
```

Comment:
This will catch cases like: [v2.8.6-debug-44347-1](https://github.com/rancher/rancher/releases/tag/untagged-baab38d19a9243ec3a06)

I don't know if this is intended, but from the comment above, it is not.